colin/core/target.py

```diff
@@ -92,3 +100,58 @@ def is_compatible(target_type, check_class, severity, tags):
     return (target_type == TargetType.DOCKERFILE and isinstance(check_class, DockerfileCheck)) \
            or (target_type == TargetType.CONTAINER and isinstance(check_class, ContainerCheck)) \
            or (target_type == TargetType.CONTAINER_IMAGE and isinstance(check_class, ImageCheck))
+
+
+class ImageName(object):
```

Comment:
This class needs tests badly.

Reply:
Done.

Do you have some other ugly image names?